src/views/system/Overview.vue

```diff
@@ -320,6 +402,100 @@ function formatDate(dateString?: string): string {
   return new Date(dateString).toLocaleDateString();
 }
 
+// Edit mode functions
+function startEditing() {
+  // Copy current data to editable state
+  editableData.value = {
+    metadata: {
+      title: systemSecurityPlan.value.metadata?.title || '',
+      version: systemSecurityPlan.value.metadata?.version || '',
+      remarks: systemSecurityPlan.value.metadata?.remarks || ''
+    },
+    systemCharacteristics: {
+      systemName: systemCharacteristics.value?.systemName || '',
+      systemNameShort: systemCharacteristics.value?.systemNameShort || '',
+      description: systemCharacteristics.value?.description || ''
+    }
+  };
+  isEditing.value = true;
```

Comment:
We should copy the entire SSP either by specifying it as a model, or by copying using ref so we don't need to recreate the whole thing here.